pontoon/base/static/js/fluent_interface.js

```diff
@@ -23,6 +23,57 @@ var Pontoon = (function (my) {
       );
   }
 
+  function renderOriginalVariant(value, elements) {
+    return '<li>' +
+      '<span class="id">' +
+        value +
+      '</span>' +
+      '<span class="value">' +
+        Pontoon.fluent.serializePlaceables(elements, true) +
+      '</span>' +
+    '</li>';
+  }
+
+  function renderOriginalSimpleElement(simpleElements, title) {
+    if (simpleElements.length) {
+      var response = renderOriginalVariant(title, simpleElements);
+      simpleElements = [];
```

Comment:
This has no effects. If your goal is to empty the content of the variable that is passed as `simpleElements`, then you'll need to do something else. As it is now, it simply makes the variable point to a different, empty array, without altering the original one. 

To clear the actual original array, you can do `simpleElements.length = 0;` or `simpleElements.splice(0, simpleElements.length);`. See https://stackoverflow.com/questions/1232040/how-do-i-empty-an-array-in-javascript

Reply:
Good point. Related to your comment below, I think we should be better off without `renderOriginalSimpleElement()`.